modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -479,4 +518,112 @@ export default class NestedTabs extends Base {
+			slider.dataset.pageX = '';
+		}
+	}
+
+	isHorizontalScroll() {
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		return slider.clientWidth < this.getChildrenWidth( slider.children ) && 'enable' === this.getHorizontalScrollSetting();
+	}
+
+	getChildrenWidth( children ) {
+		let totalWidth = 0;
+		const parentContainer = children[ 0 ].parentNode;
+		const computedStyles = getComputedStyle( parentContainer );
+		const gap = parseFloat( computedStyles.gap ) || 0; // Get the gap value or default to 0 if it's not specified
+
+		for ( let i = 0; i < children.length; i++ ) {
+			totalWidth += children[ i ].offsetWidth + gap;
+		}
+
+		return totalWidth;
+	}
+
+	setHorizontalScrollAlignment( event = {} ) {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const slider = this.elements.$headingContainer[ 0 ];
+		const tabsDirection = this.getTabsDirection();
+
+		if ( this.isHorizontalScroll() ) {
+			this.initialScrollPosition( slider, tabsDirection );
+		} else {
+			slider.style.setProperty( '--n-tabs-heading-justify-content', '' );
+		}
+	}
+
+	getTabsDirection() {
+		const currentDevice = elementorFrontend.getCurrentDeviceMode(),
+			tabsDirection = elementorFrontend.utils.controls.getResponsiveControlValue( this.getElementSettings(), 'tabs_justify_horizontal', '', currentDevice );
+
+		return tabsDirection;
+	}
+
+	initialScrollPosition( slider, tabsDirection ) {
+		switch ( tabsDirection ) {
+			// To cover defaults and center
+			case undefined:
+			case '':
+			case 'center':
+				slider.style.setProperty( '--n-tabs-heading-justify-content', 'start' );
+				slider.scrollLeft = ( this.getChildrenWidth( slider.children ) - slider.offsetWidth ) / 2;
+				break;
+			case 'end':
+				slider.style.setProperty( '--n-tabs-heading-justify-content', 'start' );
+				slider.scrollLeft = this.getChildrenWidth( slider.children );
+				break;
+			default:
+				slider.style.setProperty( '--n-tabs-heading-justify-content', '' );
+				slider.scrollLeft = 0;
+		}
+	}
+
+	getHorizontalScrollSetting() {
+		const currentDevice = elementorFrontend.getCurrentDeviceMode(),
+			horizontalScrollSetting = elementorFrontend.utils.controls.getResponsiveControlValue( this.getElementSettings(), 'horizontal_scroll', '', currentDevice );
+
+		return horizontalScrollSetting;
+	}
+
+	setHorizontalTabTitleScrollValues( event ) {
+		const slider = this.elements.$headingContainer[ 0 ],
+			isActiveScroll = slider.classList.contains( 'e-scroll' ),
+			isHorizontalScrollActive = 'enable' === this.getHorizontalScrollSetting(),
+			headingContentIsWiderThanWrapper = slider.scrollWidth > slider.clientWidth;
+
+		if ( ! isActiveScroll || ! isHorizontalScrollActive || ! headingContentIsWiderThanWrapper ) {
+			return;
+		}
+
+		event.preventDefault();
+
+		const previousPositionX = parseFloat( slider.dataset.pageX ),
+			mouseMoveX = event.pageX - previousPositionX,
+			maximumScrollValue = 5;
+
+		let toScrollDistanceX = 0;
+
+		if ( 20 < mouseMoveX ) {
```

Comment:
Why 20 is a fixed value?
I suggest placing it in a variable so we will know the intention of this value.